Core.ApplicationServices/Organizations/Write/OrganizationWriteService.cs

```diff
@@ -179,17 +179,20 @@ private Result<OrganizationMasterDataRoles, OperationError> AuthorizeAndPerformM
         var organizationDbIdMaybe = _identityResolver.ResolveDbId<Organization>(organizationUuid);
         if (organizationDbIdMaybe.IsNone) return new OperationError(OperationFailure.BadInput);
         var orgId = organizationDbIdMaybe.Value;
+        var organizationResult = _organizationService.GetOrganization(organizationUuid);
+        if (organizationResult.Failed) return new OperationError(OperationFailure.BadInput);
```

Comment:
isn't getOrganization a Result? If it is return the error from it, else remember the error message